include/model/CResourceMonitor.h

```diff
@@ -127,12 +127,18 @@ class MODEL_EXPORT CResourceMonitor {
     //! We are being told that aggressive pruning has taken place
     //! to avoid hitting the resource limit, and we should report this
     //! to the user when we can
-    void acceptPruningResult();
+    void acceptPruningStartResult();
```

Comment:
This matched what we had before, but I find these names rather strange. How about:
```suggestion
    void startPruning();
```

Reply:
`startPruning()` sort of implies that it's going to do more than maybe update the memory status.  So I renamed it but moved the debug and setting of `m_HasPruningStarted` into this method too.  And the equivalent for `endPruning()`.  Does that sound OK to you?  I am just testing it locally now.

Reply:
That sounds much better to me